src/Microsoft.TestPlatform.CrossPlatEngine/TestSession/ProxyTestSessionManager.cs

```diff
@@ -48,10 +58,17 @@ public class ProxyTestSessionManager : IProxyTestSessionManager
 
         _proxyContainerList = new List<ProxyOperationManagerContainer>();
         _proxyMap = new Dictionary<string, int>();
+        _testSessionStopwatch = new Stopwatch();
     }
 
     /// <inheritdoc/>
     public virtual bool StartSession(ITestSessionEventsHandler eventsHandler)
+    {
+        return StartSession(eventsHandler, null);
+    }
+
+    /// <inheritdoc/>
+    public virtual bool StartSession(ITestSessionEventsHandler eventsHandler, IRequestData requestData)
```

Comment:
Do we expect others to be able to override this implementation?

Reply:
Could we add that as a comment?

```suggestion
    public virtual /* for mocking purposes */ bool StartSession(ITestSessionEventsHandler eventsHandler, IRequestData requestData)
```